tincr/io/library/genFamilyInfo.tcl

```diff
@@ -38,7 +38,10 @@ proc createBlankDesignByPart { part } {
     return [tincr::designs new mydes $part]
 }
 
-# helper function used to get the relative name of Vivado elements 
+# helper functions used to get the relative name and type of Vivado elements 
```

Comment:
Separate comments for each method (`prefix` and `suffix`) providing more detail on what the method does. Also, please follow the guidelines for documenting code provided by [Doxygen](https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#tclblocks), as it will be used to generate the API documentation. For example:
```
## Get the prefix of a string
# @param s The string to split
# @param p The character(s) used to delimit the string
# @return The prefix of s up to the first instance of p
```